third_party/2and3/werkzeug/contrib/profiler.pyi

```diff
@@ -1,15 +1,11 @@
-from typing import Any, Optional
+from typing import Any, AnyStr, Iterable, Optional, Protocol
 
-available: Any
+from ..middleware.profiler import *
 
-class MergeStream:
-    streams: Any
-    def __init__(self, *streams): ...
-    def write(self, data): ...
+class _Writable(Protocol):
+    def write(self, __s: AnyStr) -> Any: ...
 
-class ProfilerMiddleware:
-    def __init__(self, app, stream: Optional[Any] = ..., sort_by=..., restrictions=..., profile_dir: Optional[Any] = ...): ...
-    def __call__(self, environ, start_response): ...
-
-def make_action(app_factory, hostname: str = ..., port: int = ..., threaded: bool = ..., processes: int = ...,
-                stream: Optional[Any] = ..., sort_by=..., restrictions=...): ...
+class MergeStream(object):
+    streams: Iterable[_Writable]
+    def __init__(self, *streams: Iterable[_Writable]) -> None: ...
```

Comment:
Should be `*streams: _Writable` because the type annotation is for individual args, not the whole thing.